tensorflow/python/distribute/cluster_resolver/kubernetes_cluster_resolver.py

```diff
@@ -23,13 +23,6 @@
 from tensorflow.python.training import server_lib
 from tensorflow.python.util.tf_export import tf_export
 
-_KUBERNETES_API_CLIENT_INSTALLED = True
-try:
-  from kubernetes import client as k8sclient  # pylint: disable=g-import-not-at-top
-  from kubernetes import config as k8sconfig  # pylint: disable=g-import-not-at-top
-except ImportError:
-  _KUBERNETES_API_CLIENT_INSTALLED = False
-
```

Comment:
It is not possible to use the `LazyLoader` class here since we rely on the import throwing immediately to set `_KUBERNETES_API_CLIENT_INSTALLED`.